consensus/istanbul/backend/engine.go

```diff
@@ -172,7 +172,14 @@ func (sb *backend) computeSignatureAddrs(header *types.Header) error {
 // given engine. Verifying the seal may be done optionally here, or explicitly
 // via the VerifySeal method.
 func (sb *backend) VerifyHeader(chain consensus.ChainReader, header *types.Header, seal bool) error {
-	return sb.verifyHeader(chain, header, nil)
+	var parent []*types.Header
+	if header.Number.Sign() == 0 {
+		// If current block is genesis, the parent is also genesis
+		parent = append(parent, chain.GetHeaderByNumber(0))
+	} else {
+		parent = append(parent, chain.GetHeader(header.ParentHash, header.Number.Uint64()-1))
+	}
+	return sb.verifyHeader(chain, header, parent)
 }
```

Comment:
May I ask why this change is needed?

Reply:
When getting validators for the startNewRound of consensus in `consensus/istanbul/core/core.go`, it calls snapshot that verify genesis block (ref. https://github.com/klaytn/klaytn/blob/dev/consensus/istanbul/backend/engine.go#L642).
- If you are asking why we need the parent as genesis in case of block number 0
  - Genesis block has no parent with hash, 0x000..., so we can use the genesis block as parent. I think it would has no problem.
- If you are asking the `else` case
  - After KIP-71 hardfork, we need the parent's header for child block's baseFee to verify that It is calculated correctly from the parent.